src/signals/incident/definitions/wizard-step-1-beschrijf.js

```diff
@@ -29,25 +36,7 @@ export default {
           className: 'col-sm-12 col-md-6',
           label: 'Hoe komt de melding binnen?',
           path: 'source',
-          values: {
-            '': 'Vul bron in',
-            'Telefoon – Adoptant': 'Telefoon – Adoptant',
-            'Telefoon – ASC': 'Telefoon – ASC',
-            'Telefoon – CCA': 'Telefoon – CCA',
-            'Telefoon - CCTR': 'Telefoon - CCTR',
-            'Telefoon – Interswitch': 'Telefoon – Interswitch',
-            'Telefoon – Stadsdeel': 'Telefoon – Stadsdeel',
-            'E-mail – CCA': 'E-mail – CCA',
-            'E-mail – ASC': 'E-mail – ASC',
-            'E-mail – Stadsdeel': 'E-mail – Stadsdeel',
-            'Webcare – CCA': 'Webcare – CCA',
-            'Eigen organisatie': 'Eigen organisatie',
-            'Meldkamer burger/ondernemer': 'Meldkamer burger/ondernemer',
-            'Meldkamer Handhaver': 'Meldkamer Handhaver',
-            'Meldkamer Politie': 'Meldkamer Politie',
-            VerbeterDeBuurt: 'VerbeterDeBuurt',
-            Waarnemingenapp: 'Waarnemingenapp',
-          },
+          values: sourceValuesObj,
```

Comment:
```js
{
  '': 'Vul bron in',
  ...sourceList,
}
```

Reply:
Not entirely; `sourceValuesObj` should be an object. The imported value is an array of objects.

Reply:
You're right! my mistake 🤦‍♂